docs/exporters/stdout.md

```diff
@@ -24,4 +28,5 @@ As always exporter's options can be displayed with `-h`:
         -V, --version    Prints version information
 
     OPTIONS:
-        -t, --timeout <timeout>    Maximum time spent measuring, in seconds. [default: 10]
\ No newline at end of file
+	    -s, --step <step_duration>    Set measurement step duration in second. [default: 2]
```

Comment:
Could you just remove the extra tabulation please ?

Reply:
Should be ok now.